src/Symfony/Component/Mime/Tests/Header/UnstructuredHeaderTest.php

```diff
@@ -244,4 +244,20 @@ public function testGetBody()
         $header = new UnstructuredHeader('Subject', 'test');
         $this->assertEquals('test', $header->getBody());
     }
+
+    public function testInReplyToIsUnstructuredHeader()
+    {
+        $email = new Email();
+        $headers = $email->getHeaders();
+        $headers->addTextHeader('In-Reply-To', 'foobar');
+        $this->assertEquals('foobar', $email->getHeaders()->get('In-Reply-To')->getBody());
+    }
+
+    public function testReferencesIsUnstructuredHeader()
```

Comment:
These tests should not be in UnstructuredHeaderTest.php, as they are not about testing the UnstructuredHeader class.
They should go in HeadersTest instead (and they should instantiate the Headers object directly, as done by other tests there instead of using an Email object just to get the Headers object from it)